app/static/src/js/helpers.js

```diff
@@ -48,7 +48,7 @@ if (showTagsElement) {
 
 // This is a temporary fix to ensure tab changing works for flag history.
 // Updating to `4.7.0` for govuk css broke this functionality. (v0.0.252)
-// Upgrading `govuk-frontend-jinja==2.7.0` (the equivalent 4.7.0 version) does not fix it.
+// Upgrading `govuk-frontend-jinja==2.8.0` (the equivalent 4.7.0 version) does not fix it.
```

Comment:
@ramsharma-prog I don't think we need these comments anymore as we bumped to 2,8. Check with whoever made these comments in the code before removing it.

Reply:
Good point! removed @rayhan15243

Reply:
![image](https://github.com/communitiesuk/funding-service-design-assessment/assets/36962596/1b3beb15-34b7-4eaa-9a55-78978cf1dbe1)
@ramsharma-prog

Reply:
2.8 working as expected. Checked on Dev.